### ISSUE TITLE
[Feature] Reorder photos within albums

### ISSUE THREAD

**Issue by andrewmc201** [leonlatsch/Photok#385]
The ability to sort photos within albums is a reasonably simple solution to a couple of other issues listed in this project. It would also allow the user to customize how their album is displayed, as well.

I'd like to be able to press a button to enable reordering mode, or hold and drag images to sort.

**Comment by leonlatsch** [comment]
Good idea. Generally I want to add the feature of sorting (album or all files). A custom sorting is of course something useful
